README.md

```diff
@@ -11,6 +14,8 @@ All the source is from the original author. We just modified it such that it run
 
 Technokrat GmbH takes no credit whatsoever on the original codebase.
 
+ciao si il corso e molto interessante
```

Comment:
I think this got forgotten :) Please remove this :)

Reply:
done